dogonlanguages/templates/other.mako

```diff
@@ -84,13 +77,19 @@
+<p>In 2012 we discovered that competent informants were still present in Nyafogo, and we began emergency salvage fieldwork. This was initially carried out by Abbie Hantgan. We also found that a Burkinabé linguistics student at the University of Ouagadougou, Aminata Ouattara, was also working on Tiefo-N. She is an ethnic Tiefo but not a native speaker. There were some hitches in the fieldwork due to Hantgan's departure for a post-doc at SOAS in January 2013, and Ouattara's taking a year off to work for an NGO. In 2016 Heath and Ouattara teamed up to do intensive work with the last two fluent Tiefo-N speakers at our base in Bobo Dioulasso with some trips to Nyafogo village. See links below for references. Ouattara defended her master’s thesis on Tiefo-N in early 2019.</p>
+
+<p> In the period 2017-2019 Heath and Ouattara have also done extensive work on Tiefo-D, which is still spoken in some of the quartiers of Daramandougou village cluster. There are four dialects, the most divergent being that of Biton quartier. A grammar and lexical spreadsheets are at an advanced stage as of summer 2019 and will likely be finished in early summer 2020.</p>
+
+<p>A distinctive feature of Tiefo-D grammar is the use of motion verbs 'come' and 'go' in multi-clause constructions. Both verbs occur in constructions of the type 'came and VPed' and 'went and VPed', and elaborations like 'came and came and VPed' and 'went and went and VPed', and even 'came and went and VPed' (not involving round-trip motion), where the motion verbs function to highlight events, with or without involving actual motion. The second motion verb in such constructions is contracted or suppletive.</p>
+
+<p>Funding for the Tiefo-N project was from NSF, and that for the Tiefo-D project is from NEH.</p>
 
 <p>Click on the links for drafts of our own work on Tiefo-N, and English/French adaptations/summaries of Winkelmann's German-language work.</p>
 
 <ul>
-<li>${h.external_link('https://github.com/clld/dogonlanguages-data/raw/master/beta/literature/Heath_Ouattara_Hantgan_Short_Grammar_Tiefo-N.docx', label='Heath_Ouattara_Hantgan_Short_Grammar_Tiefo-N.docx')}</li>
-<li>${h.external_link('https://github.com/clld/dogonlanguages-data/raw/master/beta/literature/Heath_Ouattara_Hantgan_Tiefo-N_lexicon_spreadsheet.xlsx', label='Heath_Ouattara_Hantgan_Tiefo-N_lexicon_spreadsheet.xlsx')}</li>
+<li>${h.external_link('http://dx.doi.org/10.17617/2.2378140', label='Short grammar of Tiefo-N of Nyafogo (Gur, Burkina Faso)')}</li>
```

Comment:
The currently recommended URL representation of a DOI is `https://doi.org/<DOI>` - although, `http://dx.doi.org` still seems to work.

Reply:
Go figure, no one told "Deep Blue":

https://deepblue.lib.umich.edu/handle/2027.42/139025

I'll update them and push.

Reply:
Yeah, to some extent all these "persistent identifiers" are a bit of a joke. In particular when they use the "we never said the URL representation is persistent" excuse ...